packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx

```diff
@@ -232,6 +245,9 @@ const mapDispatchToProps = (dispatch: Dispatch, props: IFullProps) => {
         variables.user.id = variables.user._fhirID
         delete variables.user._fhirID
       }
+      if (variables.user.signature) {
+        delete variables.user.signature.__typename
```

Comment:
This also should not be needed as I didn't find __typename inside signature.

Reply:
Modified the code. Thanks for inform.